.github/workflows/error-ref-publisher.yaml

```diff
@@ -0,0 +1,63 @@
+name: Meshery-Istio Error Codes Utility Runner
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '**.go'
+
+jobs:
+  Update-error-codes:
+    name: Error codes utility
+    if: github.repository == 'meshery/meshery-istio'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        # token here with write access to meshery-istio repo
+        with:
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          ref: 'master'
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ secrets.GO_VERSION }}
+
+      - name: Run utility
+        run: |
+          go run github.com/layer5io/meshkit/cmd/errorutil -d . update --skip-dirs meshery -i ./helpers -o ./helpers
```

Comment:
also, I noticed that we have another line from meshkit here, is it desireable? @DelusionalOptimist

Reply:
Good point. Actually this command here fetches the utility from meshkit. Let's keep this.